node/sc/bridge_manager.go

```diff
@@ -520,8 +519,8 @@ func (bm *BridgeManager) LogBridgeStatus() {
 		return
 	}
 
-	p2cTotalRequestNonce, p2cTotalHandleNonce := uint64(0), uint64(0)
-	c2pTotalRequestNonce, c2pTotalHandleNonce := uint64(0), uint64(0)
+	p2cTotalRequestNonce, p2cTotalHandleNonce, p2cTotalLowerHandleNonce := uint64(0), uint64(0), uint64(0)
```

Comment:
How about simply define `var uint64`? All the variables will be initialized to zero value by default.

Reply:
reflect.